package.json

```
@@ -19,17 +19,18 @@
     ]
   },
   "dependencies": {
-    "@hapi/hoek": "9.x.x",
-    "@hapi/b64": "5.x.x",
-    "@hapi/boom": "9.x.x",
-    "@hapi/bourne": "2.x.x",
-    "@hapi/cryptiles": "5.x.x"
+    "@hapi/hoek": "^10.0.0",
+    "@hapi/b64": "^5.0.0",
```

Pending https://github.com/hapijs/b64/pull/40
```suggestion
    "@hapi/b64": "^6.0.0",
```